ingestion/functions/retrieval/retrieval.py

```diff
@@ -1,19 +1,66 @@
-import json
+import boto3
```

Comment:
is that missing from the requirements.txt?

Reply:
requirements.txt is everything _besides_ the AWS SDK (which is included in the lambda execution environment by default). It's conceivable one might want to pin a version, but it doesn't seem common practice.

Reply:
ah ok thanks for the clarification!